docs/teams/kickstart.md

```diff
@@ -0,0 +1,25 @@
+# Kickstart
```

Comment:
Maybe on this page also link to the general Kickstart page?

Reply:
There isn't one. There's a page on reviewing kickstarts, but no page on kickstarts themselves.

(There definitely _should_ be one, but that's OOS)